cmake/std/atdm/sems-rhel7/environment.sh

```diff
@@ -147,9 +147,12 @@ elif [[ "$ATDM_CONFIG_COMPILER" == "INTEL-17.0.1" ]] ; then
   export OMPI_CXX=`which icpc`
   export OMPI_CC=`which icc`
   export OMPI_FC=`which ifort`
-  export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$SEMS_INTEL_ROOT/mkl/lib/intel64/
-  export ATDM_CONFIG_LAPACK_LIBS="-mkl"
-  export ATDM_CONFIG_BLAS_LIBS="-mkl"
+#  export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$SEMS_INTEL_ROOT/mkl/lib/intel64/
+#  export ATDM_CONFIG_LAPACK_LIBS="-mkl"
+#  export ATDM_CONFIG_BLAS_LIBS="-mkl"
+  export LAPACK_ROOT=/usr/lib64/atlas
+  export ATDM_CONFIG_LAPACK_LIBS="-L${LAPACK_ROOT};-llapack"
+  export ATDM_CONFIG_BLAS_LIBS="-L${BLAS_ROOT}/lib;-lblas"
```

Comment:
@jmgate and @bathmatt,

 I did a quick check of this after the merge to 'develop' and there is no `BLAS_ROOT` var defined so this evaluates to:

```
export ATDM_CONFIG_BLAS_LIBS="-L/lib;-lblas"
```

which is not correct.  But there is no `/lib/libblas.*` file, at least not on the CEE LAN machine 'ascicgpu15' so not sure what this does.

I will attempt to fix this in my next PR where I address some other stuff as well.

Reply:
Another problem.  I the machine 'ascicgpu15', there is no directory `/usr/lib64/atlas` as shown by:

```
[rabartl@ascicgpu15 CHECKIN]$ ls /usr/lib64/atlas
ls: cannot access /usr/lib64/atlas: No such file or directory
```

I tested this with:

```
./checkin-test-atdm.sh intel-openmp-shared-debug --enable-packages=Kokkos,Teuchos --local-do-all
```

and it passed:

```
PASSED (NOT READY TO PUSH): Trilinos: ascicgpu15

Wed Mar 20 07:56:08 MDT 2019

Enabled Packages: Kokkos, Teuchos

Build test results:
-------------------
1) intel-openmp-shared-debug => passed: passed=164,notpassed=0 (2.33 min)
```

My guess is that the compiler is just ignoring `-L/usr/lib64/atlas` and is using the default system BLAS and LAPACK on that machine:

```
/usr/lib64/libblas.so
/usr/lib64/libblas.so.3
/usr/lib64/libblas.so.3.4
/usr/lib64/libblas.so.3.4.2
/usr/lib64/liblapack.so
/usr/lib64/liblapack.so.3
/usr/lib64/liblapack.so.3.4
/usr/lib64/liblapack.so.3.4.2
```

So we are getting lucky that it seems to be falling back on default system blas/lapack and it seems to work (at least on the machine 'ascicgpu15' where the only 'sems-rhel7' testing is being done).